design-toolkit/components/src/components/table/index.tsx

```diff
@@ -13,20 +13,19 @@
 'use client';
 import 'client-only';
 
-import { ArrowDown, ArrowUp, Kebab } from '@accelint/icons';
+import { Kebab } from '@accelint/icons';
 import Pin from '@accelint/icons/pin';
```

Comment:
Try and be consistent about either merging imports or having them all be specific